### ISSUE TITLE
Bot can't see all visits

### ISSUE THREAD

**Issue by Pr3tend3r** [dyrkin/luxmed-bot#36]
Hi,

First of all I wanted to thank you for your work :)

Unfortunately, I noticed that the bot does not see all doctors.
Although many of them are in the search option the bot does not see free appointments with them and monitoring does not work on them. 

![screen_luxmed_all](https://user-images.githubusercontent.com/100491855/178155272-b57f7832-cb75-49cc-9322-38da8849f384.JPG)

![scrren_luxmed](https://user-images.githubusercontent.com/100491855/178155277-a8be7758-5808-4775-ab32-521c7e1c9506.JPG)

Could you please help with this issue?
Thank you in advance.

**Comment by dyrkin** [comment]
Hi,

This is because the website, the app, and the bot, all don't allow you to search for more than 14 days in a row. In the bot, you've chosen a 2 months period, but the bot actually has looked for appointments in the period from 10 till 24 of July.
On the website, you've chosen a 14 days period starting from the 24th of July, that's why you see several appointments from August.
Select the same period in the bot and you'll see the same list of appointments.

**Comment by Pr3tend3r** [comment]
Hi,

Thank you for the explanation now I understand.

Following this line of thought, monitoring also works only 12 days ahead so if there is a free doctor's appointment e.g. 20 days ahead the bot will not book it because it can only see 12 days into the future. The bot will have to wait for the appointment to be within the range of these 12 days.

Do I understand correctly?

**Comment by Pr3tend3r** [comment]
Sorry, I accidentally closed the topic

**Comment by dyrkin** [comment]
Yeah, that's correct. I might be fixing this by sending multiple requests with different ranges to the server but I would prefer not to ddos it) 
I can introduce an option to enable sending of multiple requests. For example, if you choose the range: 10.07 - 15.08 it will send 3 requests: 10.07 - 24.07, 25.07 -  09.08, 10.08-15.08. Then it will collect results into a single response and send it to the bot. But it'll be possible to enable it on your local instance of the bot, not on this shared one.

**Comment by Pr3tend3r** [comment]
You are right, a large number of requests can be seen as DDoS. I think 14 days is completely enough. 

Thank you for your help and support :)
